e2e/cypress/integration/enterprise/integrations/incoming_webhook_spec.js

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @enterprise @incoming_webhook
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
```

Comment:
Failed on me. (btw, we'll find time to fix this in the future)
```suggestion
import * as TIMEOUTS from '../../../fixtures/timeouts';
```